packages/cli-lib/projectsWatch.js

```diff
@@ -186,7 +186,16 @@ const createWatcher = async (
       projectConfig.name,
       projectSourceDir,
       filePath,
-      'delete'
+      'deleteFile'
+    );
+  });
+  watcher.on('unlinkDir', async filePath => {
+    addFile(
```

Comment:
Not a blocker, but this func supports folders and also the "delete" action now. I wonder if we should change the name of it to be more generic.